bin/commands/cmd_gen.ml

```diff
@@ -19,7 +19,7 @@ let rec atdgen ?(rescript = false) ?output_prefix:original_prefix = function
     let* _, _, version = Generator.split_elements file in
     let output_prefix =
       Option.map
-        (fun output_prefix -> output_prefix ^ "_" ^ string_of_int version)
+        (fun output_prefix -> output_prefix ^ "_" ^ version)
```

Comment:
Now, this is the question, are we allowing both string and number or just string from now on? I wouldn't have anything against supporting just strings honestly.

Reply:
I think just strings, it makes sense... but I think we should give a warning if the atd files are in an order which doesn't make sense, like if the user gives us `stork gen file_2.atd file_1.atd` we should warn them?

Reply:
We could, but it could be a follow up improvement/good first issue.

Reply:
string being in that context a totally ordered superset of integers, I think it's alright to just support strings.

Reply:
as discussed with @naartjie, we actually have to support both string or int, otherwise we won't be able to make it work with our own files @wokalski